src/Cache_Command.php

```diff
@@ -20,34 +18,130 @@ class Cache_Command extends WP_CLI_Command {
 	 *
 	 * ## OPTIONS
 	 *
-	 * <post_id>
+	 * [<post_id>]
 	 * : ID of the post to clear the cache for.
 	 *
+	 * [--all]
+	 * : Clear all oEmbed caches.
+	 *
 	 * ## EXAMPLES
 	 *
 	 *     # Clear cache for a post
 	 *     $ wp embed cache clear 123
 	 *     Success: Cleared oEmbed cache.
```

Comment:
Let's change the message to include the post ID.
```suggestion
	 *     Success: Cleared oEmbed cache for post 123.
```